chess/6-gameplay/gameplay.md

```diff
@@ -104,7 +104,7 @@ classDiagram
 
 ```
 
-The important thing is that along with the fields that are generally required for a `UserGameCommand`, that there is also an additional field named `move` that contains the fields of a `ChessMove` object in your JSON serialization.
+In the end, the important thing is that you design supports the additional `move` field when serializing the `MAKE_MOVE` command over the WebSocket. This must looks something like the following:
```

Comment:
This is great!

There are two adjustments:
* Fix "you design" to "your design"
* Fix "This must looks something like..."
    * Possibility 1: "This looks something like..."
    * Possibility 2: "This must look like..."

Reply:
Oh my. I obviously need some sleep. Thanks for catching this.

Reply:
You're doing great. Thanks for all your hard work 😄